package.json

```diff
@@ -73,8 +73,7 @@
     "yaml": "^2.2.2",
     "semver": "^7.5.2",
     "optionator": "~0.9.3",
-    "typescript": "~5.1.6",
-    "@cypress/request": "^3.0.0"
```

Comment:
`@cypress/request` is no longer needed?

Reply:
this is an override that was previously needed to fix npm audit